code/go/0chain.net/blobbercore/writemarker/worker.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/allocation"
 	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/config"
 	"github.com/0chain/blobber/code/go/0chain.net/blobbercore/datastore"
+	"github.com/0chain/blobber/code/go/0chain.net/core/logging"
```

Comment:
Duplicate logging package import, perhaps remove the below one

Reply:
fixed